src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -413,15 +415,15 @@ def _yangConvert(val):
         return vValue
 
     """
-    Xlate a Qos Maps list
+    Xlate a Type 1 map list
     This function will xlate from a dict in config DB to a Yang JSON list
     using yang model. Output will be go in self.xlateJson
 
     Note: Exceptions from this function are collected in exceptionList and
     are displayed only when an entry is not xlated properly from ConfigDB
     to sonic_yang.json.
 
-    QOS MAPS Yang has inner list, which is diffrent from config DB.
+    Type 1 map Yang has inner list, which is diffrent from config DB.
```

Comment:
Type 1 Lists has inner list, which is different from generic lists in config DB.

Reply:
Will update, thanks